File: javascript/challenge19.js
const pumpkinSpice = money => {
    let pies = Math.floor(money/5)
    let remainder = money%5;
    let lattes = Math.floor(remainder/3);
    remainder = remainder%3;
    let macarons = remainder
    let grams = pies*30+lattes*15+macarons*3
    return [pies, lattes, macarons, grams];
}

File: ruby/challenge19.rb
def pumpkinSpice(money)
    pies = (money / 5).floor
    r = money % 5
    l = floor(r / 3).floor
    m = r % 3
    g = 30 * pies + 15 * l + 3 * m
    return [pies, l, m, g]
end
